ComputerVision/SDK/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,236 @@
+            foreach (var obj in analysis.Objects)
+            {
+                Console.WriteLine("{0} with confidence {1} at location {2},{3},{4},{5}",
+                    obj.ObjectProperty, obj.Confidence,
+                    obj.Rectangle.X, obj.Rectangle.X + obj.Rectangle.W,
+                    obj.Rectangle.Y, obj.Rectangle.Y + obj.Rectangle.H);
+            }
+            Console.WriteLine("\n");
+        }
+
+        private static void DisplayBrandDetectionResults(ImageAnalysis analysis)
+        {
+            //brands
+            Console.WriteLine("Brands:");
+            foreach (var brand in analysis.Brands)
+            {
+                Console.WriteLine("Logo of {0} with confidence {1} at location {2},{3},{4},{5}",
+                    brand.Name, brand.Confidence,
+                    brand.Rectangle.X, brand.Rectangle.X + brand.Rectangle.W,
+                    brand.Rectangle.Y, brand.Rectangle.Y + brand.Rectangle.H);
+            }
+            Console.WriteLine("\n");
+        }
+
+        private static void DisplayDomainSpecificResults(ImageAnalysis analysis)
+        {
+            //celebrities
+            Console.WriteLine("Celebrities:");
+            foreach (var category in analysis.Categories)
+            {
+                if (category.Detail?.Celebrities != null)
+                {
+                    foreach (var celeb in category.Detail.Celebrities)
+                    {
+                        Console.WriteLine("Name: {0} with confidence {1} at location {2},{3},{4},{5}" , 
+                            celeb.Name, celeb.Confidence,
+                            celeb.FaceRectangle.Left, celeb.FaceRectangle.Top,
+                            celeb.FaceRectangle.Height, celeb.FaceRectangle.Width);
+                    }
+                }
+            }
+
+            //landmarks
+            Console.WriteLine("Landmarks:");
+            foreach (var category in analysis.Categories)
+            {
+                if (category.Detail?.Landmarks != null)
+                {
+                    foreach (var landmark in category.Detail.Landmarks)
+                    {
+                        Console.WriteLine("Name: {0} with confidence {1}" ,landmark.Name, landmark.Confidence);
+                    }
+                }                
+            }
+            Console.WriteLine("\n");
+        }
+
+        private static void DisplayColorSchemeResults(ImageAnalysis analysis)
+        {
+            //color scheme
+            Console.WriteLine("Color Scheme:");
+            Console.WriteLine("Is black and white?:" + analysis.Color.IsBWImg);
+            Console.WriteLine("Accent color:" + analysis.Color.AccentColor);
+            Console.WriteLine("Dominant background color:" + analysis.Color.DominantColorBackground);
+            Console.WriteLine("Dominant foreground color:" + analysis.Color.DominantColorForeground);
+            Console.WriteLine("Dominant colors:");
```

Comment:
It will be less verbose to do the individual colors in a single line, separated by comma. You can also do it without a loop, like this:
Console.WriteLine("Dominant colors: {0}", string.Join(", ", analysis.Color.DominantColors));